GameData/RealismOverhaul/Engine_Configs/AJ10_137_Config.cfg

```diff
@@ -134,18 +134,20 @@
 //Apollo 13: 1 ignition, 0 failures
 //Apollo 15,16,17 (J-class): 7 ignitions, 0 failures
 //Skylab 2,3,4: unkown, 3 ignitions assumed
-//ASTP: Unkown, 3 ignitions assumed
-//17 engines flow with 78 ignitions, no failures
+//ASTP: Unknown, 3 ignitions assumed
+//17 ignitions, 0 failures
+//17 cycles, 0 failures
+//61 restarts, 0 failures
 @PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[AJ10-137]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
 {
 	TESTFLIGHT
 	{
 		name = AJ10-137
 		ratedBurnTime = 750		//appears to be ablative, no extra burn time - [A] describes it as 12.5min "max"
-		ignitionReliabilityStart = 0.987342
-		ignitionReliabilityEnd = 0.997468
-		cycleReliabilityStart = 0.944444
-		cycleReliabilityEnd = 0.998889
+		ignitionReliabilityStart = 0.990506
+		ignitionReliabilityEnd = 0.998101
+		cycleReliabilityStart = 0.958333
+		cycleReliabilityEnd = 0.991667
```

Comment:
A minimum failure chance of 1% per run seems very, very high for SPS, given the amount of work that went into making it as reliable as possible. EDIT: in particular, it has a lower ceiling than the AJ10-138 on Transtage, which *actually suffered failures*. I would be inclined to give it at least four nines for reliability end in both ignition and cycle, though start values being low-ish seem fine.

Reply:
Same here, forgot to give the SPS special treatment